src/tweezer.js

```diff
@@ -29,43 +30,44 @@ export default class Tweezer {
   }
 
   on (name, handler) {
-    this.events[name] = this.events[name] || []
-    this.events[name].push(handler)
+    const e = this.events[name]
+    if (e) e.push(handler)
+    else this.events[name] = [handler]
     return this
   }
 
   emit (name, val) {
-    let e = this.events[name]
-    e && e.forEach(handler => handler.call(this, val))
+    const e = this.events[name]
+    if (e) e.forEach(handler => handler.call(this, val))
+    return this
```

Comment:
Emit is really more of a private method for the class and I'd like to keep it that way.